docs/noise.md

```diff
@@ -0,0 +1,121 @@
+## Noise
+
+For simulation it is useful to have `Gate`s that enact noisy quantum evolution.
+Currently Cirq supports modeling noise via *operator sum* representations of
+noise (these evolutions are also known as quantum operations or quantum
+dynamical maps).  This formalism models evolution of the density matrix
+via
+
+![Operator sum representation: $\rho \rightarrow \sum_k A_k \rho A_k^\dagger$](resources/OperatorSumDef.gif)
+
+Where here the the A<sub>k</sub> are *Krauss* operators. These operators
+are not necessarily unitary and must satisfy the trace preserving
+property
```

Comment:
A natural question a reader may have at this point is whether Krauss operators are unique  for a given quantum operation (it's easy to see they're not).

A second question is then what is the minimum number of terms in operator sum representation (it's a little more work to see that it's d^2 where d is the dimension of the underlying Hilbert space).

Reply:
True, though if one interprets the index k as a measurement outcome via the Stinespring dilation this matters.  We'll be adding the ability for channels that support keys to be measurements in simulations, so in that case it would matter.

I've updated this to say they are not unique but pointed to John Preskill's notes for more details.

Reply:
Makes sense. Thanks for the reference!